css/DayPickerKeyboardShortcuts.scss

```diff
@@ -0,0 +1,106 @@
+.DayPickerKeyboardShortcuts__show,
+.DayPickerKeyboardShortcuts__close {
+  background: none;
+  border: 0;
+  color: inherit;
+  font: inherit;
+  line-height: normal;
+  overflow: visible;
+  padding: 0;
+  cursor: pointer;
+
+  &:active {
+    outline: none;
+  }
+}
+
+.DayPickerKeyboardShortcuts__show {
+  border-top: 26px solid $react-dates-color-white;
+  border-right: 33px solid $react-dates-color-primary;
+  width: 22px;
+  position: absolute;
+  right: 0;
+  bottom: 0;
+
+  &:hover {
+    border-right: 33px solid #008489;
+  }
+}
+
+.DayPickerKeyboardShortcuts__show_span {
+  color: $react-dates-color-white;
+  position: absolute;
+  bottom: 0;
+  right: -28px;
+}
+
+.DayPickerKeyboardShortcuts__panel {
+  background: $react-dates-color-white;
+  border: 1px solid $react-dates-color-border;
+  border-radius: 2px;
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  left: 0;
+  z-index: 2;
+  padding: 22px;
+  margin: 33px;
+}
+
+.DayPickerKeyboardShortcuts__title {
+  font-size: 16px;
+  font-weight: bold;
+  margin: 0;
+}
+
+.DayPickerKeyboardShortcuts__list {
+  list-style: none;
+  padding: 0;
+
+}
+
+.DayPickerKeyboardShortcuts__close {
+  position: absolute;
+  right: 22px;
+  top: 22px;
+  z-index: 2;
+
+  svg {
+    height: 15px;
+    width: 15px;
+    fill: $react-dates-color-gray-lighter;
+
+    &:hover,
+    &:focus {
+      fill: $react-dates-color-gray-light;
```

Comment:
perhaps something to do in a future update, but one thing we came across during a11y testing is that users who have ie9 and enable a setting that essentially turns off background colors and text colors will end up not being able to notice the difference here. We'll likely need to do something where we change the border color or add an outline to the element for more visibility.